.github/workflows/hello-reason.yml

```diff
@@ -1,14 +1,20 @@
 name: Hello Reason
 
-on: [push, pull_request]
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
```

Comment:
That's not right, the CI should be executed on all branches, and I used of this in the past to test stuff without making noise on the master